data-model/annotations/0.1.0/examples/example-annotation-event-for-web.json

```diff
@@ -0,0 +1,36 @@
+{
+  "data": {
+    "type": "annotation",
+    "attributes": {
+      "annotations": [
+        {
+          "oa:motivation": "oa:commenting",
+          "oa:target": {
+            "ods:id": "https://hdl.handle.net/20.500.125/1111111",
+            "ods:type": "https://doi.org/21.T11148/894b1e6cad57e921764e",
+            "oa:selector": {
+              "ods:type": "ClassSelector",
+              "oa:class": "specimenName"
+            }
+          },
+          "oa:body": {
+            "ods:type": "body",
+            "oa:value": [],
+            "dcterms:reference": "reference",
+            "ods:score": 0.9
+          }
+        }
+      ],
+      "batchMetadata": [
+        {
+          "searchParams": [
+            {
+              "inputField": "field",
+              "inputValue": "value"
```

Comment:
What is input field and what is input value?

Reply:
added examples, see the description of these two fields above:
input field: "JsonPath (mixed notation) of field used to base annotation reasoning on. Indexes must be wildcards"
input value: "Value stored at the field indicated in inputField"

Reply:
value of the object. 

say you have:
```
inputField = occurrences[*].weather.temperature
inputValue = 20 C

```
Maybe you want to annotate the field `occurrences[*].weather.condition` with the value "that sounds like a nice day!"

So basically, input field/value is saying, "because this field is equal to this value. I am making my annotations thusly." and then we'd look for all specimen with `occurrences[*].weather.temperature = 20 C`, and make that annotation

Reply:
Not quite. input field/value is not the same as the annotation target. **It defines the search criteria used to find the objects we want to make the batch annotations on.**

Having the search param:

```
searchParams: 
[{
  inputField: occurrences[*].weather.temperature
  inputValue: ""
}]

```

Would mean that we find all specimens where temperature is empty. Maybe then you make the following annotation on ALL specimens with an empty temperature field: 

```
annotation: {
   target: occurrences[*].weather.condition
   body: I wonder what the weather was like!
}
```

It may be possible that the annotation target field and the input field for the search params are the same. Like if you want to annotate a missing value or something. but it's helpful to think of them as completely separate concepts.

Reply:
Ah ok, so the search params really exist to find other specimens which should also be annotated within the batch.
That makes sense. The name convention of the properties is very plain so that's why I was wondering.